weave-js/src/components/PagePanelComponents/Home/Browse3.tsx

```diff
@@ -308,115 +311,119 @@ const MainPeekingLayout: FC = () => {
   useMousetrap('esc', closePeek);
 
   return (
-    <Box
-      sx={{
-        flex: '1 1 auto',
-        width: '100%',
-        height: '100%',
-        display: 'flex',
-        overflow: 'hidden',
-        flexDirection: 'row',
-        alignContent: 'stretch',
-      }}>
+    <WFDataModelAutoProvider
```

Comment:
While this looks like 214 lines of code changes, it is just one wrapper I added to the top here `WFDataModelAutoProvider`. Which automatically detects if the project is backed by Streamtable or Clickhouse, and provides the correct interface for the calling components.